### PR TITLE
Adds "quit" formula

### PR DESCRIPTION
**Pull request by theunraveler**
This PR adds a formula for the `quit` utility: http://www.macupdate.com/app/mac/31821/quit.

### CODE REVIEW

**Comment by DomT4** [comment]
```
==> Verifying quit-1.9.2.zip checksum
Error: SHA256 mismatch
Expected: 2845a3ff0078e7ddbd349a88ada640e038226e261bed399546791e328dfe5ef5
Actual: 6d4f41e69d2477cb3d2e7bb51996f0f13ce6edcbf6e86d2c7e64094d2bfd35be
Archive: /Library/Caches/Homebrew/quit-1.9.2.zip
```

**Comment by theunraveler** [comment]
Fixed. Not sure why the SHA was off--I used `brew create` to create the formula.

**Comment by dunn** [comment]
Is this being actively developed? I can't see on MacUpdate when the last release was.

**Comment by DomT4** [comment]
The checksum has changed again in the mean time, so we'll probably need to find another source of download if we are going to include this.

```
~> gsha256sum quit.1.9.2.zip
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  quit.1.9.2.zip
```

**Comment by theunraveler** [comment]
Yes, it is actively updated--last release on 22-Sep-2015.

Regarding the changing checksum, is this a problem that formula maintainers have encountered before? Any recommendations? Should I get in touch with the developer of the script and ask them?

**Comment by DomT4** [comment]
We have no other formulae that fetch from `macupdate`, although perhaps this sort of thing is why, heh. Perhaps `macupdate` forcefully regenerates the `zip` file for some reason or such. It may be worth notifying the author of the package that hosting it elsewhere (even as a mirror) would help package this.

File: Library/Formula/quit.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Quit < Formula
+  desc "Command-line tool to politely quit applications"
+  homepage "http://www.macupdate.com/app/mac/31821/quit"
+  url "http://www.macupdate.com/download/31821/quit.1.9.2.zip"
+  sha256 "6d4f41e69d2477cb3d2e7bb51996f0f13ce6edcbf6e86d2c7e64094d2bfd35be"
```
Comment (bfontaine) [inline Library/Formula/quit.rb]: You can use `https://` for both these URLs.

**Comment by apjanke** [comment]
That Macupdate URL just redirects to a user Dropbox account for me; I don't think they're actually hosting it.

```
$ wget http://www.macupdate.com/download/31821/quit.1.9.2.zip
--2015-11-11 01:06:35--  http://www.macupdate.com/download/31821/quit.1.9.2.zip
Resolving www.macupdate.com... 64.188.61.169
Connecting to www.macupdate.com|64.188.61.169|:80... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: http://dl.dropbox.com/u/258627/Software/quit.1.9.2.zip [following]
--2015-11-11 01:06:36--  http://dl.dropbox.com/u/258627/Software/quit.1.9.2.zip
```

The version I downloaded has the same `6d4f41e69d2477cb3d2e7bb51996f0f13ce6edcbf6e86d2c7e64094d2bfd35be` SHA256 as Dom's earlier download, and the README inside says the last update was "Sept 17, 2015".  The zip file also includes AppleDouble format Finder info files. Could be the author has re-published it with minor changes without changing the same version, or Macupdate might be round-robining its redirects. Neither situation is great. Would be nice to have a different hosting source for it.

**Comment by apjanke** [comment]
Contact for author: http://jon.stovell.info/software/

**Comment by theunraveler** [comment]
Contacted the developer on Oct 3, no response yet.

**Comment by DomT4** [comment]
Going to close this one out for now, pending developer response to your contact. If they haven't responded in ~5 weeks, we tend to find they often don't tend to indefinitely :cry:.

In combination with the checksum moving around we can't carry this at the moment. Thank you for the submission though, and please feel free to resubmit if you can persuade the developer to host or mirror on Github or Bintray or such.

They're independent so I can't promise anything at all, but you may have better luck submitting this to the [Caskroom](https://github.com/caskroom/homebrew-cask) who often don't bother with checksums, whereas core Homebrew enforces them.

**Pull request closed without merging**